### PR TITLE
Addbasic elements

### PR DESCRIPTION
**Pull request by Dygerydoo**
Edited classes, and added used icons on design to find them more quickly.

image

### CODE REVIEW

**Comment by 007lva** [comment]
:+1:

File: app/views/psique/documentation/elements.html.haml [hunk 1]
```diff
@@ -19,184 +19,257 @@
...
+    %i.fa.fa-thumb-tack fa-thumb-tack
+  .col-3.icon-margin
+    %i.fa.fa-thumbs-o-up fa-thumbs-o-up
+
+  .col-4.icon-margin
+    %i.fa.fa-trash-o fa-trash-o
+  .col-4.icon-margin
+    %i.fa.fa-user fa-user
+  .col-3.icon-margin
+    %i.fa.fa-thumbs-o-up fa-thumbs-o-up
```
Comment (scruti) [inline app/views/psique/documentation/elements.html.haml]: Missing new line at the end of file.

**Comment by gponsu** [comment]
+1 I need some buttons of this PR. Add new line at the end of file  and merge please.

**Comment by scruti** [comment]
:+1:

**Pull request merged**